cypress/integration/ix-video/poster.spec.ts

```diff
@@ -18,8 +18,12 @@ context('ix-video: poster', () => {
     it('should append width/height params to poster URL equal to video size', () => {
       cy.get(ixVideoTag).then(($ixVideo) => {
         const videoTag = $ixVideo.find('[part=video]');
-        const videoTagWith = videoTag.css('width').split('px')[0];
-        const videoTagHeight = videoTag.css('height').split('px')[0];
+        const videoTagWith = parseFloat(
+          videoTag.css('width').split('px')[0]
+        ).toFixed(0);
```

Comment:
Just to confirm: it was the test's expectation that width/height return a long float, not the component logic itself?

Reply:
That's correct. The `.should()` in line 27 "expected" is a long float if but the "actual" is not a long float.

The funny thing is this wasn't an issue for a long time. I have no idea why cypress is all of a sudden giving us floats here when evaluating `videoTag.css`.